### ISSUE TITLE
[feature request] to load external coordinates into a PDB, MOL2, ...

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#795]
Hi, I got that if 
```python
pmd.load_file('polyAT.pdb', xyz='polyAT.pdb')
```

Do you think it's worth to make consistent API for this
```python
pmd.load_file('polyAT.prmtop', xyz='polyAT.rst7')
```

**Comment by swails** [comment]
In this case yes, but only because I can see a use-case for this.

```python
pmd.load_file('polyAT.pdb', xyz='polyAT.pdb')
```

is not a use-case :smiley:.  However,

```python
pmd.load_file('polyAT.pdb', xyz='polyAT.minimized.rst7')
```

is a legitimate use-case, if you want to use the PDB as a source of topology and atom information rather than coordinates.  The "consistency" argument is not particularly compelling to me.

To me, this is a feature request to load external coordinates into a PDB.

**Comment by hainm** [comment]
> pmd.load_file('polyAT.pdb', xyz='polyAT.minimized.rst7')

yes, that is. (I was too lazy to find an rst7 file for my example).

**Comment by hainm** [comment]
update title.
